content/en/docs/concepts/policy/pod-security-policy.md

```diff
@@ -493,8 +493,8 @@ recommended with this strategy.
 
 - *MustRunAs* - Requires at least one `range` to be specified. Uses the
 minimum value of the first range as the default. Validates against all ranges.
-- *MustRunAsNonRoot* - Requires that the pod be submitted with a non-zero
-`runAsUser` or have the `USER` directive defined (using a numeric GID) in the
+- *MayRunAs* - Does not Require that RunAsGroup be specified. However, when RunAsGroup are 
+specified, they have to fall in the defined range.
 image. No default provided. Setting `allowPrivilegeEscalation=false` is strongly
```

Comment:
It looks like  "image. No default provided." was left in by accident. Can you remove this